src/components/Collection/Sidebar.css

```diff
@@ -3,7 +3,7 @@
   width: 250px;
   padding: 8px 0 12px;
   position: fixed;
-  max-height: calc(100vh -168px);
+  max-height: calc(100vh - 168px);
```

Comment:
Just wondering what the `168px` is based off of -- it seems to be a lot of extra padding (vs something like `113px`).

Reply:
It is based in the page margin. --pageMargin: 84px 18px; and set it to top margin plus bottom margin.

Reply:
we could use something like: calc(100vh - var(--topBarHeight) * 2); giving us 112px

Reply:
@igarbla I think top margin + bottom margin is the best idea -- 84+18 = 102px (instead of 168px).

Reply:
Well, in fact, top margin + bottom margin = 168px; it could be top-margin + font-size + line-height:
calc(100vh - 84px - 1.5rem)

Reply:
Oops, my bad. 168px may be the best option then, if it would overlap the bottom footer if it was less. Otherwise, you could do 84px+28px (top margin plus the standard margin at the bottom of the other headers). Up to you, I'm good either way.

Reply:
i'm fine with 112px.